vllm/v1/metrics/loggers.py

```diff
@@ -652,13 +652,18 @@ def __init__(
         engine_idxs: Optional[list[int]] = None,
         custom_stat_loggers: Optional[list[StatLoggerFactory]] = None,
         enable_default_loggers: bool = True,
+        client_count: int = 1,
     ):
         self.engine_idxs = engine_idxs if engine_idxs else [0]
 
         factories: list[StatLoggerFactory] = []
         if custom_stat_loggers is not None:
             factories.extend(custom_stat_loggers)
-
+        if client_count > 1:
```

Comment:
@njhill Would it be better to use `len(self.engine_idxs) > 1` here?

Reply:
No because that's the number of DP engines and it's fine for it to be > 1 if there's only one API server.